src/components/bs5/header/header.hbs

```diff
@@ -702,6 +702,240 @@
                 </a>
             </div>
             {{/if}}
+            {{#if siteHeader.config.endorsed_enabled}}
+            <div class="qld__header__brand">
+                <a href="{{siteHeader.url.value}}">
+                    <div class="qld__header__brand-image qld__header__brand-image_subtype">
+                        {{#if siteHeader.logo.value}}
+                        <img src="{{siteHeader.logo.value}}" width="170" height="56">
+                        <title>Queensland Government</title>
+                        </img>
+                        {{else}}
+                        <svg width="170" height="56" viewBox="0 0 170 56" fill="none" xmlns="http://www.w3.org/2000/svg"
```

Comment:
could this be a file that is embedded on dist build?

Reply:
Yes I also thought maybe converting to inline SVG, with the child <image tag> containing a base64 encoded image as per this example:
https://github.com/qld-gov-au/qgds-qol-mvp/blob/019ccd39d7969f8c08b89448204d6a7136b35828/src/assets/img/endorsed.svg#L7 

its there incase editor doesn't set correct config, will fall back to COA